src/features/deposit/components/DepositUIMachineProvider.tsx

```diff
@@ -240,12 +240,14 @@ export function DepositUIMachineProvider({
                 )
 
                 const tx = createDepositFromSiloTransaction(
-                  tokenAddress,
+                  tokenAddress === "native"
```

Comment:
`tokenAddress` won't be `native` in the future.

Reply:
noted